script/releases

```diff
@@ -14,24 +16,40 @@ github.authenticate({
 
 console.log('Fetching electron/electron releases from the GitHub API')
 
-async function getReleases () {
-  const res = await github.repos.getReleases({
+function ghOpts(opts) {
+  const defaults = {
     owner: 'electron',
     repo: 'electron',
     per_page: 100
-  })
+  }
+  return Object.assign({}, defaults, opts)
+}
+
+async function getReleases () {
 
-  return Promise.all(res.data
+  // determine number of pages
+  const countRes = await github.repos.getReleases(
+    Object.assign(ghOpts({per_page: 1}))
+  )
+  const count = Number(parseLinkHeader(countRes.meta.link).last.page)
+  const pagesToFetch = Math.ceil(count / 100)
+  
+  // paginate
+  let releases = []
+  for (let i = 1; i <= pagesToFetch; i++) {
+    const batch = await await github.repos.getReleases(ghOpts({page: i}))
```

Comment:
`await await`?

Reply:
Ha! Super-patient. Miraculously, that worked. Fixed now.